vagrant/devmode/salt/roots/virtualenv.sls

```diff
@@ -29,6 +29,7 @@ pip_pkgs:
     - activate: true
     - requirements: /home/vagrant/calamari/requirements/2.7/requirements.txt
     - download_cache: /vagrant/pip_cache
+    - env_vars: SWIG_FEATURES=-cpperraswarn
```

Comment:
Yep, that was one of my first comments.  It's a big hammer.  Oh well.  The right thing to do is unbundle all this crap and make it the distros' problem anyway.